lib/manager/cargo/types.ts

```diff
@@ -12,6 +13,19 @@ export interface CargoSection {
   'build-dependencies': CargoDeps;
 }
 
-export interface CargoConfig extends CargoSection {
+export interface CargoManifest extends CargoSection {
```

Comment:
`Cargo.toml` is actually the cargo manifest, `.cargo/config[.toml]` is the cargo config - this rename fixes that.